src/main/java/org/apache/sling/distribution/journal/kafka/KafkaClientProvider.java

```diff
@@ -112,7 +115,7 @@ public void close() {
     @Override
     public Closeable createPoller(String topicName, Reset reset, @Nullable String assign, HandlerAdapter<?>... adapters) {
         String consumerGroupId = UUID.randomUUID().toString();
-        KafkaConsumer<String, byte[]> consumer = new KafkaConsumer<>(consumerConfig(ByteArrayDeserializer.class, consumerGroupId, reset));
+        KafkaConsumer<String, byte[]> consumer = createConsumer(ByteArrayDeserializer.class, reset);
```

Comment:
Due to this refactoring the `info` level log statement below is wrong (the `consumerGroupId` is not the one used in the message poller).

Reply:
Thanks .. will fix it

Reply:
thanks @cschneider !